SinkLogger.py

```diff
@@ -57,7 +57,7 @@ def registerExtenderCallbacks(self, callbacks):
 
         # pattern: replacement passed into re.sub()
         self.sinkPatterns = {
-            r'\.innerHTML=([^=])': r'.innerHTML=QF9iYXlvdG9w.innerHTML=\1',
+            r'\.innerHTML(|[ ])=([^=])': r'.innerHTML=QF9iYXlvdG9w.innerHTML=\2',
```

Comment:
```suggestion
            r'\.innerHTML\s*=([^=])': r'.innerHTML=QF9iYXlvdG9w.innerHTML=\1',
```

This will match all whitespace instead of just a single space.